### PR TITLE
[feat] add post process for enu because of using bpe model

### PR DESCRIPTION
**Pull request by chwma0**

### CODE REVIEW

**Review by robin1001 — CHANGES_REQUESTED** [review]
please see inline

File: runtime/core/decoder/torch_asr_decoder.cc [hunk 1]
```diff
@@ -214,6 +214,39 @@ void TorchAsrDecoder::AttentionRescoring() {
       result_ = result;
     }
   }
+
+  if (opts_.bpe_decode && result_.size()) {
```
Comment (robin1001) [inline runtime/core/decoder/torch_asr_decoder.cc:218]: Make it a function?

File: runtime/core/decoder/torch_asr_decoder.cc [hunk 2]
```diff
@@ -214,6 +214,39 @@ void TorchAsrDecoder::AttentionRescoring() {
...
+    using std::string;
+    using std::vector;
+    vector<string> characters;
+
+    if (SplitUTF8String(result_, &characters)) {
+      string result;
+      const string delim = "\xe2\x96\x81";
```
Comment (robin1001) [inline runtime/core/decoder/torch_asr_decoder.cc:225]: Make it a global constant variable in utils.h, kBlankSymbol?

Comment (chwma0) [reply runtime/core/decoder/torch_asr_decoder.cc:225]: OK

File: runtime/core/decoder/torch_asr_decoder.h [hunk 3]
```diff
@@ -46,6 +47,7 @@ class TorchAsrDecoder {
   // Return true if we reach the end of the feature pipeline
   bool AdvanceDecoding();
   void AttentionRescoring();
+  void PostProcesser();
```
Comment (robin1001) [inline runtime/core/decoder/torch_asr_decoder.h:50]: change to ProcessBlank()?  and add some comment and example here, such as
// Replace special blank  symbol "▁" (U+2581, \xe2\x96\x81) with white space.
PostProcesser is more than just just blank processing.

File: runtime/core/decoder/torch_asr_decoder.h [hunk 4]
```diff
@@ -24,6 +24,7 @@ using TorchModule = torch::jit::script::Module;
 struct DecodeOptions {
   int chunk_size = 16;
   int num_left_chunks = -1;
+  bool bpe_decode = true;
```
Comment (robin1001) [inline runtime/core/decoder/torch_asr_decoder.h:27]: bpe_decode is not required here, we also use ▁ for white space in character based training.

**Review by robin1001 — CHANGES_REQUESTED** [review]
please see inline.

**Review by robin1001 — APPROVED** [review]

**Pull request merged**